# numpy.linalg.LinAlgError

exception linalg.LinAlgError[source]

    
Generic Python-exception-derived object raised by linalg functions.

General purpose exception class, derived from Python’s exception.Exception
class, programmatically raised in linalg functions when a Linear Algebra-
related condition would prevent further correct execution of the function.

Parameters

    
None

#### Examples

    
    >>> from numpy import linalg as LA
    >>> LA.inv(np.zeros((2,2)))
    Traceback (most recent call last):
      File "<stdin>", line 1, in <module>
      File "...linalg.py", line 350,
        in inv return wrap(solve(a, identity(a.shape[0], dtype=a.dtype)))
      File "...linalg.py", line 249,
        in solve
        raise LinAlgError('Singular matrix')
    numpy.linalg.LinAlgError: Singular matrix
    
© 2005–2021 NumPy Developers  
Licensed under the 3-clause BSD License.  
https://numpy.org/doc/1.22/reference/generated/numpy.linalg.LinAlgError.html

  *[ISP]: Internet Service Provider
  *[LIFO]: last-in, first-out
  *[FIFO]: first-in, first-out

